webui/react/src/utils/metric.ts

```diff
@@ -55,7 +55,7 @@ export const extractMetricSortValue = (
 };
 
 export const extractMetricValue = (workload: WorkloadGroup, metric: Metric): number | undefined => {
-  const source = workload.metrics[metric.group]?.metrics ?? {};
+  const source = workload.metrics?.[metric.group]?.metrics ?? {};
```

Comment:
`workload.metric` type is 
https://github.com/determined-ai/determined/blob/0ead6aca08914c57d63e77c85b45fd4ceaff01c5/webui/react/src/types.ts#L498

so added optional chaining to check if the key exists in `workload.metric`